chainer/functions/activation/clipped_relu.py

```diff
@@ -155,3 +155,36 @@ def clipped_relu(x, z=20.0):
     """
     y, = ClippedReLU(z).apply((x,))
     return y
+
+
+def relu6(x):
+    """Rectifier Unit function clipped at 6.
+
+    It computes
+
+    .. math:: \\text{ReLU6}(x) = \\min(\\max(0, x), 6).
+
+    Args:
+        x (:class:`~chainer.Variable` or :ref:`ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    Returns:
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
```

Comment:
A hyperlink could help users who want other thresholds.
```suggestion

    .. seealso:: :func:`chainer.functions.clipped_relu`

```